src/porting.cpp

```diff
@@ -697,6 +701,26 @@ int mt_snprintf(char *buf, const size_t buf_size, const char *fmt, ...)
 	return c;
 }
 
+bool openURL(std::string url)
+{
+	if (!std::regex_match(url,
+			std::regex(R"(^https?:\/\/([\w\.-]+)\.([a-z\.]{2,6})([\/\w+% \.-]*)*(\?[\w=]*)?\/?$)") )) {
+		errorstream << "Invalid url: " << url << std::endl;
+		return false;
+	}
+
+#if defined(_WIN32)
+	return system((std::string("start '' '") + url + "'").c_str()) == 0;
```

Comment:
and `ShellExecuteA` here
https://stackoverflow.com/questions/17347950/#answer-17348170